requirements-dev.txt

```diff
@@ -1,3 +1,10 @@
+# required
+Cython>=0.16
+fiona>=1.7
+pandas>=0.23.4
+pyproj>=1.9.3
+shapely>=1.5
```

Comment:
I updated the versions of a few to to the minimal requirements we're currently using in CI, but I didn't check all the version requirements closely./